multihost_test/Bugzillas/data/bz_667593_1.sh

```diff
@@ -0,0 +1,21 @@
+function sgTestPass() {
```

Comment:
Would it be possible to merge the scripts in this file, bz_667593_2.sh, bz_667593_3.sh and bz_667593_4.sh in one?

Reply:
it is not possible to merge this functions to one file then call them . So i am putting them in different file and calling them

Reply:
How come? bz_667593_1.sh and bz_667593_3.sh are exactly the same file. bz_667593_1.sh and bz_667593_2.sh just differ in the password value from line 9, and you can parametrize that. bz_667593_1.sh and bz_667593_4.sh are different, so I agree that you can't merge them in one file so easily.

Reply:
Done